lib/util.js

```diff
@@ -885,6 +887,13 @@ function formatRaw(ctx, value, recurseTimes) {
   }
   ctx.seen.pop();
 
+  if (ctx.sorted && extrasType === kObjectType) {
+    if (ctx.sorted === true) {
+      output = output.sort();
+      return reduceToSingleString(ctx, output.sort(), base, braces);
+    }
+    return reduceToSingleString(ctx, output.sort(ctx.sorted), base, braces);
+  }
   return reduceToSingleString(ctx, output, base, braces);
```

Comment:
👆 To dry out the condition above it could be something like:
```js
const comparator = ctx.sorted === true ? undefined : ctx.sorted
return reduceToSingleString(ctx, output.sort(comparator), base, braces)
```